{{cookiecutter.repository_name}}/operations/evaluate.py

```diff
@@ -38,16 +40,14 @@ def evaluate(config):
     }
 
     for name, data_loader in evaluate_data_loaders.items():
-        for examples, standardized_images in tqdm(data_loader, desc=name, leave=False):
+        for standardized_images, examples in lantern.ProgressBar(data_loader, name):
             with lantern.module_eval(model):
                 predictions = model.predictions(standardized_images)
-                loss = predictions.loss(examples)
 
-            evaluate_metrics[name]["loss"].update_(loss)
-            evaluate_metrics[name]["accuracy"].update_(examples, predictions)
+            evaluate_metrics[name]["standard"].update_(predictions, examples)
 
         for metric_name, metric in evaluate_metrics[name].items():
-            metric.log(tensorboard_logger, name, metric_name)
+            metric.log_dict(tensorboard_logger, name)
```

Comment:
This looks wrong, you are writing the name to metrics?
If it is correct, maybe better naming would be good so it's easy to read